content/chat/rooms/moderation/hive-dashboard.textile

```diff
@@ -9,4 +9,45 @@ languages:
   - kotlin
 ---
 
-Hive dashboard documentation to go here.
+"Hive Dashboard":https://hivemoderation.com/dashboard is a powerful all-in-one moderation tool, which allows you to set up rules to combine automated AI moderation with human review.
+
+The Hive Dashboard rule is a rule applied to chat rooms which allows you to use Hive's "moderation dashboard":https://docs.thehive.ai/docs/what-is-the-moderation-dashboard to review messages in your chat room.
+
+h2(#fields). Rule fields
+
+- Hive API Key := The API key for your hive account.
```

Comment:
This API key is specific to the dashboard product, so we'd probably want to reference that